package.json

```diff
@@ -1,6 +1,7 @@
 {
   "dependencies": {
-    "express": "^4.18.2"
+    "express": "^4.18.2",
+    "node.js": "^0.0.1-security"
```

Comment:
Why do we need  this `node.js` package ?
If its not used anywhere then we need to remove it